scripts/build-package.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# The envirinment set by create-ubuntu-environmens sscript is trusty, so we should match it.
+# Look at debian/gbp.conf for default optiuons sent to gbp.
+rm -r debian/tmp debian/*.log
+git commit -a -m 'Preparing for pbuild' && git push
```

Comment:
That seems like a dangerous/annoying piece of code to sneak in here. Why is it necessary to create a new commit?